Formula/soccerwindow2.rb

```diff
@@ -16,12 +16,17 @@ class Soccerwindow2 < Formula
   depends_on "librcsc"
 
   def install
+    # ld: framework not found Security-lz
+    inreplace "configure",
```

Comment:
Don't you just love it when projects share buggy code and it ends up everywhere.